### ISSUE TITLE
use koji repos for testing

### ISSUE THREAD

**Issue by ignatenkobrain** [CentOS-PaaS-SIG/ci-pipeline#688]
Right now we use just fedora+updates(+updates-testing)? for released fedora and just rawhide for rawhide.

This means that if you update multiple packages simultaneously, your test pipeline will fail because your dependency is not yet in compose.

**Comment by ignatenkobrain** [comment]
>  The code adds a repo for the rpms built for that build. So if you are testing sed, you get a repo added for all the rpms from that sed build. That's it. There is no updates-testing or anything like that enabled. Whatever comes by default from Fedora is there, that's it.

Yes, this is what I mean by `fedora+updates(+updates-testing)?`

Basically those repos do not contain all latest builds which make tests fail if there was no compose yet.

**Comment by johnbieren** [comment]
Right, it sounded like (although I didn't follow closely) Miroslav and you were coming to a solution for this.
